sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/ca/DefaultCAClient.java

```diff
@@ -110,8 +104,8 @@ public X509Certificate sign(byte[] csr) throws Exception {
         return certificates;
     }
 
-    private JSONObject doPostRequest(String path, JSONObject data) throws IOException, CAException {
-        Request request = Request.Post(URLUtils.addPathUnchecked(baseURL, path).toString())
+    private JSONObject doPostRequest(String path, JSONObject data) throws IOException, CAException, URISyntaxException {
+        Request request = Request.Post(new URI(DcosConstants.CA_BASE_URI + path))
```

Comment:
@mhrabovcin do we lose anything making this a constant? Given that unit tests pass, I assume we weren't doing any actual network tests of this?

Reply:
I was using this originally to test this class against local cluster when I was developing it originally. It is not used anymore.